v1/openapi/schemas/jet-drops.yaml

```diff
@@ -9,6 +9,7 @@ properties:
     example: 100
   result:
     type: array
+    description: Array of entries. Size depends on the `limit` filtering parameter.
```

Comment:
Size of what? How does it depend? Limit is a pagination parameter.

Suggestion: Array of entries. The `limit` pagination parameter sets the number of entries.

Reply:
thx